war/src/main/scss/modules/buttons-deprecated.scss

```diff
@@ -1,8 +1,6 @@
-@import url("../abstracts/theme.less");
-
 // Button: variant mixins
 
-.button-danger() {
+@mixin button-danger {
```

Comment:
Something seems to be going wrong in this file. The following styles are missing from the generated CSS in the SCSS version, while they are present in the Less version:

```css
.button-link {
  color: var(--primary);
  background-color: transparent;
  border-color: transparent;
  font-weight: var(--btn-link-font-weight);
}
.button-link:hover,
.button-link:focus {
  color: var(--btn-link-color--hover);
  background-color: var(--btn-link-bg--hover);
  border-color: var(--btn-link-bg--hover);
}
.button-link:focus {
  box-shadow: none;
}
.button-link:active {
  color: var(--btn-link-color--active);
  background-color: var(--btn-link-bg--active);
  border-color: var(--btn-link-bg--active);
}
.button-disabled {
  opacity: 0.5;
  pointer-events: none;
}
```

Reply:
Good spot, restored.